### PR TITLE
Cart page

### PR DESCRIPTION
**Pull request by RakshithBhat03**
Added cart page for e-commerce.
Consists of the following:
- Cart section
- Coupon section
- Checkout and summary section

Preview: https://deploy-preview-3--clutchstore.netlify.app/pages/cart.html

### CODE REVIEW

**Review by nikhilmalviya-dev — APPROVED** [review]
code is fine

File: css/cart.css [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+@import url(./footer.css);
+
+.wrapper {
+  background-color: var(--color-extra-light-gray);
+}
+
+.cart-wrapper {
+  grid-template-columns: 2fr 1fr;
+  grid-template-areas:
+    "product-cart product-coupon"
```
Comment (bajajcodes) [inline css/cart.css:11]: Good grid-area names are self-describing and meaningful.

File: css/cart.css [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+.product-summary {
+  grid-area: product-summary;
+  max-height: calc(23 * var(--size-9));
+  align-self: start;
+}
+
+/* Product-cart */
+
+.card__body {
```
Comment (bajajcodes) [inline css/cart.css:39]: - Class names here can be made consistent, commonly and majorly you have used is `-`.

File: css/cart.css [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+  border-radius: var(--radius-lg);
+  font-size: calc(1.15 * var(--size-9));
+  text-transform: uppercase;
+  color: var(--primary-color-dark);
+}
+
+@media screen and (max-width: 1300px) {
+  .cart-wrapper {
+    grid-template-columns: 1fr;
+    grid-template-areas:
```
Comment (bajajcodes) [inline css/cart.css:78]: > Here your going to single-column layout, I guess and by default your UI is on multi-column layout, then how is this getting applied.

**Review by bajajcodes — COMMENTED** [review]
- Code is written clean and self-descriptive :+1:

File: css/cart.css (hunk 2 above)
Comment (RakshithBhat03) [reply css/cart.css:39]: This was used to overwrite the css of card component

File: css/cart.css (hunk 3 above)
Comment (RakshithBhat03) [reply css/cart.css:78]: It is getting applied to a single column only. I have used `grid-template-areas` to order the containers.

**Pull request merged**